version.php

```diff
@@ -29,8 +29,8 @@
     die('Direct access to this script is forbidden.'); // It must be included from a Moodle page.
 }
 
-$plugin->version   = 2021011911;      // The current plugin version (Date: YYYYMMDDXX).
-$plugin->release   = 2021011910;      // Same as version
+$plugin->version   = 2021122110;      // The current plugin version (Date: YYYYMMDDXX).
+$plugin->release   = 2021122110;      // Same as version
 $plugin->requires  = 2014051200;      // Requires Moodle 2.7 or later.
 $plugin->component = "local_envbar";
```

Comment:
```suggestion
$plugin->component = 'local_envbar';
```
Out of scope but since you're touching this file, might as well fix this up as well.